### ISSUE TITLE
Tie 2d and 3d boxs

### ISSUE THREAD

**Issue by ZPavlo** [waymo-research/waymo-open-dataset#23]
Hi, can you explain how to tie 2d and 3d boxes?

**Comment by peisun1115** [comment]
Did you get this resolved? 

Just in case, we do not provide direct correspondence between 2d/3d boxes. That is an interesting task to solve though. One way to get this correspondence naively is to compare the projected 3d boxes and the camera 2d boxes.

projected 3d box: https://github.com/waymo-research/waymo-open-dataset/blob/master/waymo_open_dataset/dataset.proto#L290
camera 2d box: https://github.com/waymo-research/waymo-open-dataset/blob/master/waymo_open_dataset/dataset.proto#L295

**Comment by kevinsu628** [comment]
Hi @peisun1115 I have a question about the 2D&3D bboxes. 
I have downloaded the first 4 training zips, and I found that all 4 zips have 2D boxes labelled in camera_labels. In projected_lidar_labels, only x, y labelled. 

My question is: Waymo mentions in the download page that, "All segments contain 3D labels. The 100 segments that also contain 2D labels have been packaged into the first three files under 'Training' and the first file under 'Validation'."
Shouldn't the labels in projected_lidar_labels also have z coordinates? Why can I still find 2D labels in the 4th zip if only the first 3 files have 2D bbox in camera_labels?
